packages/core/components/Modal/Modal.tsx

```diff
@@ -218,7 +220,7 @@ const ModalRenderer = (
   const [menuOpened, setMenuOpened] = React.useState('');
 
   React.useEffect(() => {
-    addWindows({ icon, title });
+    addWindows({ icon, title, hasButton: hasWindowButton || true });
```

Comment:
I think this will never work: when `hasWindowButton` is falsy, `addWindows` will receive `true` instead :(


To make it works we could do this:
```suggestion
    addWindows({ icon, title, hasButton: hasWindowButton });
```

And also change this [line](https://github.com/React95/React95/pull/227/files#diff-203a31eb04ac2d1337bc09f4ddc77105a9397898999f05703ccd8346b1a5e779R199) adding a default value to `hasWindowButton = true`

What do you think?

Reply:
Oh yes sure, silly me.  I should have wrapped hasWindowButton with a isDefined() function at least.
Initially I wanted to do what you describe here, but my IDE complains with the following message : 

![Capture d’écran 2021-02-09 à 14 03 02](https://user-images.githubusercontent.com/16033537/107367513-adbc9280-6adf-11eb-85c2-294fef20432a.png)

I do believe it's because I left a question mark in ModalProps, so hasWindowButton is not mandatory as a prop - hence, the IDE believes that this value might be undefined, regardless what's written in the defaultProps.
Typical typescript / defaultProps conflict here.